core/src/main/java/hivemall/smile/tools/TreeExportUDF.java

```diff
@@ -141,17 +141,17 @@ public String getDisplayString(String[] children) {
     }
 
     public enum OutputType {
-        javascript, graphvis;
+        javascript, graphviz;
 
         @Nonnull
         public static OutputType resolve(@Nonnull String name) throws UDFArgumentException {
             if ("js".equalsIgnoreCase(name) || "javascript".equalsIgnoreCase(name)) {
                 return javascript;
-            } else if ("dot".equalsIgnoreCase(name) || "graphvis".equalsIgnoreCase(name)) {
-                return graphvis;
+            } else if ("dot".equalsIgnoreCase(name) || "graphviz".equalsIgnoreCase(name)) {
```

Comment:
better to have `"graphvis".equalsIgnoreCase(name)` for backward compatibility.

Reply:
`if("dot".equalsIgnoreCase(name) || "graphviz".equalsIgnoreCase(name) || "graphvis".equalsIgnoreCase(name))`

Reply:
c299bd3355ae1e640f12978c76ffb8d5a912d951